.github/workflows/ci.yml

```
@@ -57,4 +57,8 @@
       - name: Upload coverage data to CodeScene
         if: env.CS_ACCESS_TOKEN
         run: cs-coverage upload --format "lcov" --metric "line-coverage" "lcov.info"
+      - name: Upload coverage data to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)